db/seeds.rb

```diff
@@ -5,3 +5,16 @@
 #
 #   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
 #   Mayor.create(name: 'Emanuel', city: cities.first)
+require 'open-uri'
+
+page = Nokogiri::HTML(open("http://www.languagedaily.com/learn-german/vocabulary/most-common-german-words-2"))
+
+lines = page.css("table tr:not(:first-child)")
+
+lines.each do |line|
+  original_text = line.css("td:nth-child(2)").text
+  translated_text = line.css("td:nth-child(3)").text
+  Card.create(original_text: original_text, translated_text: translated_text, review_date: Date.today + 3.days)
```

Comment:
Line is too long. [111/80]<br>Do not use `Date.today` without zone. Use `Time.zone.today` instead.